README.md

```diff
@@ -11,13 +11,20 @@ The Zig logo and mascots are licensed under the Attribution-ShareAlike 4.0 Inter
 
 <img alt="Zig Icon" src="zig-icon.svg" width="300">
 
-zig-icon.svg is an icon of just the graphical part of the logo.
+`zig-icon.svg` is an icon of just the graphical part of the logo.
 
 ## Favicon
 
 <img alt="Zig Favicon" src="zig-favicon.png" width="16" height="16">
 
-zig-icon.png is a small version of the icon suitable for a website favicon.
+`zig-icon.png` is a small version of the icon made for website favicons.
+
+# Logo versions
+
+* `full logo/zig_color_dark.svg` is the main logo, preferably used on applicable backgrounds.
+* `full logo/zig_color_white.svg` when visibility of the main logotype ("ZIG") is too low.
+* `full logo/zig_dark.svg` where color reproduction is not possible, or visibility of colored symbol is too low.
+* `full logo/zig_white.svg` where color reproduction is not possible, or visibility of `zig_dark.svg` is too low.
```

Comment:
Pardon this possibly ignorant question- is there any reason to have these versions in this repository of canonical forms, since they can be trivially generated from either of the color versions by changing the color to black/white?

Reply:
To answer your first question, they are 100% identical. I will have to rename the index logo to make it consistent and logical with the colour versions.

To cater for differences in versions, the naming could be something like 
`zig_logo_black.svg` for colour symbol, black text.
`zig_logo_white.svg` for colour symbol, white text.
`zig_logo_black.nev.svg` for black symbol, black text.
`zig_logo_white.nev.svg` for white symbol, white text.
`zig_symbol_color.svg` for colour symbol.
`zig_symbol_white.nev.svg` for white symbol.
`zig_symbol_black.nev.svg` for black symbol.

Second question, there's two reasons you'd have these versions.

1. If for print, white/light version is sometimes adjusted slightly to be "thicker" some places to make up for ink flow tightening margins, much like smaller sizes on screen. This reason is not applicable here for web.

2. Second reason is to have easily accessible versions for people that do not have access to nor knowledge of working with SVG icons. If you feel like this is not a concern, you can remove the folders containing copies in various colour combinations.